docs/current_docs/features/visualization.mdx

```diff
@@ -11,9 +11,9 @@ This quickly creates the need to visualize your pipelines to understand and meas
 
 ## Terminal UI
 
-The [Dagger CLI](../reference/cli/) includes a real-time visualization feature called the terminal UI (TUI). It shows a full-blown DAG in a style similar to `git log --graph`, with the current state of your DAG's evaluation and a full snapshot of all output at the end.
+The [Dagger CLI](../api/cli/) includes a real-time visualization feature called the terminal UI (TUI). It shows a full-blown DAG in a style similar to `git log --graph`, with the current state of your DAG's evaluation and a full snapshot of all output at the end.
```

Comment:
```suggestion
The [Dagger CLI](../api/cli.mdx) includes a real-time visualization feature called the terminal UI (TUI). It shows a full-blown DAG in a style similar to `git log --graph`, with the current state of your DAG's evaluation and a full snapshot of all output at the end.
```

Reply:
IMO better to link to the file than the URL, because the URL can be changed in doc metadata and then a broken link to the URL will not trigger an error, but a broken link to a file will.